adminapi/urls.py

```diff
@@ -1,12 +1,17 @@
 from django.conf.urls import patterns, url
+from django.views.generic import TemplateView
 
 from adminapi.views import LoginView, UserListView, TestView
 
 
 urlpatterns = patterns(
-
     "",
-    url(r"^users/$", UserListView.as_view(), name="user-list"),
-    url(r"^login-auth/$", LoginView.as_view(), name="admin-login-auth"),
-    url(r"^test-view/$", TestView.as_view(), name="test-view"),
+    # SPA template views
+    url("^login/app/$", TemplateView.as_view(template_name="react/react_login.html"), name="app-login"),
```

Comment:
The naming will suffice for now. I'll create an issue where we namespace the API and react bits so the names and URLs can be cleaner.